packages/x-components/src/components/result/__tests__/base-result-add-to-cart.spec.ts

```diff
@@ -1,47 +1,86 @@
-import { mount } from '@vue/test-utils';
+import { Result } from '@empathyco/x-types';
+import { mount, Wrapper } from '@vue/test-utils';
 import { getResultsStub } from '../../../__stubs__/results-stubs.factory';
 import { getDataTestSelector, installNewXPlugin } from '../../../__tests__/utils';
+import { XBus } from '../../../plugins/index';
+import { Dictionary } from '../../../utils/index';
 import BaseResultAddToCart from '../base-result-add-to-cart.vue';
 
 describe('testing BaseResultAddToCart component', () => {
   const result = getResultsStub()[0];
-  const [, localVue] = installNewXPlugin();
 
-  it('emits UserClickedResultAddToCart when the user click on the component', () => {
-    const resultAddToCartWrapper = mount(BaseResultAddToCart, {
-      localVue,
-      propsData: { result }
-    });
+  function renderAddToCart({
+    result = { name: 'TEST' } as Result,
```

Comment:
You are always rendering the same result. Why not set its default value here and then return it in the API object? That way you can also remove the global result, but you still have the rendered result available for when you need to access it.